src/js/tech/html5.js

```diff
@@ -282,6 +293,33 @@ class Html5 extends Tech {
     this.textTracks().removeTrack_(e.track);
   }
 
+  removeOldTextTracks() {
```

Comment:
Hmmm I kinda copied this from the `handle...` functions. Why are they not postfixed with `_` but this one should be? As far as I know they actually should all be underscored?